sleap_io/io/video.py

```diff
@@ -189,6 +192,16 @@ def frames(self) -> int:
         """Number of frames in the video."""
         return self.shape[0]
 
+    @property
+    def frame_rate(self) -> Optional[float]:
+        """Frames per second of the video."""
+        video_extensions = ["mp4", "avi", "mov", "mj2", "mkv", "slp"]
+        if isinstance(self.filename, str) and any(
+            self.filename.endswith(ext) for ext in video_extensions
+        ):
+            return cv2.VideoCapture(self.filename).get(cv2.CAP_PROP_FPS)
```

Comment:
SUBGROUP REVIEW: We don't need to check the video extension or filename here since this property of frame_rate would only ever be accessed if filename pointed to a VIdeo object anyways